frontend/requests/requestDialogService.js

```diff
@@ -82,18 +64,37 @@
             });
         };
 
+        function selectDialogToShow(request, event, dialogProperties) {
+            let requestType = request.type_of_invite;
+            requestType = request.isStatusOn('sent') ? requestType : RESOLVED_REQUEST;
+            requestType = request.areInstitutionsValid() ? requestType : INVALID_REQUEST;
+
+            switch(requestType) {
+                case RESOLVED_REQUEST:
+                    service.showResolvedReqDialog(event); break;
+                case INVALID_REQUEST:
+                    service.showInvalidReqDialog(event); break;
+                case REQUEST_PARENT:
```

Comment:
Why is it empty? should have at least one break.

Reply:
It is empty because both cases have the same behavior and it is not possible to do 
case REQUEST_PARENT || REQUEST_CHILDREN